base/abstractarraymath.jl

```diff
@@ -61,20 +61,20 @@ julia> a = reshape(Vector(1:4),(2,2,1,1))
  1  3
  2  4
 
-julia> squeeze(a; dims=3)
+julia> dropdims(a; dims=3)
 2×2×1 Array{Int64,3}:
 [:, :, 1] =
  1  3
  2  4
 ```
 """
-squeeze(A; dims) = _squeeze(A, dims)
-function _squeeze(A::AbstractArray, dims::Dims)
+dropdims(A; dims) = _dropdims(A, dims)
```

Comment:
Not strictly related to this PR, but what's the value of having dims be a keyword argument for a simple two arg function? I know they are fast now, but here it doesn't seem needed. The rename makes this a bit more jarring, since we now have `dropdims(A,dims=1)` which is redundant.

I believe there are other functions where I've noticed this, though can't remember which exactly. I can open an issue if we want.

Reply:
It's just for consistency --- all functions that accept a list of dimensions accept them with a `dims` keyword argument, so it's easier to guess or remember how to call it.

Reply:
This was done for consistency. All functions that operate on dimensions now take those as a keyword argument.

Reply:
jinx!

Reply:
Yes I see now they were all changed. I'm not a huge fan, but I suppose that ship has sailed.